packages/swingset-runner/demo/exchangeBenchmark/exchanger.js

```diff
@@ -13,22 +13,17 @@ const log = makePrintLog();
  * @param {ZoeService} zoe
  * @param {Issuer[]} issuers
  * @param {Payment[]} payments
- * @param {Record<string,Installation>} installations
+ * @param {{ makeInvitation: () => Invitation }} publicAPI
```

Comment:
for consistency, please rename `publicAPI` to `publicFacet`

Reply:
Will do.